app/views/admin/attachments/_form.html.erb

```diff
@@ -1,12 +1,28 @@
 <% initialise_script "GOVUK.AdminAttachmentsForm", selector: '.js-attachment-form', right_to_left_locales: Locale.right_to_left.collect(&:to_param) %>
 
 <%= form_for attachment, url: [:admin, typecast_for_attachable_routing(attachable), attachment.becomes(Attachment)], as: :attachment, html: { :class => "js-attachment-form" } do |form| %>
-  <%= form.errors %>
-
-  <%= content_tag :fieldset, class: ("right-to-left" if form.object.rtl_locale?) do %>
-    <%= form.text_field :title %>
+  <% if form.object.errors.any? %>
+    <%= render "govuk_publishing_components/components/error_summary", {
+      title: "To save the #{form.object.class.name.demodulize.underscore.humanize.downcase} please fix the following issues:",
+      items: form.object.errors.full_messages.map do |msg| {
+          text: msg
+        }
+      end
+    } %>
   <% end %>
 
+  <div class="govuk-!-margin-bottom-8">
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: "Title (required)"
+      },
+      name: "attachment[title]",
+      heading_level: 2,
+      heading_size: "l",
+      value: form.object.title.blank? ? "" : form.object.title
```

Comment:
I think you can just pass in `form.object.title` and it will handle the empty in the component.

```suggestion
      value: form.object.title
```

Reply:
👍